eng/native/gen-buildsys.sh

```diff
@@ -95,6 +95,9 @@ fi
 
 if [[ "$host_arch" == "wasm" ]]; then
     if [[ "$target_os" == "browser" ]]; then
+        if [[ -n "$NATIVEAOT_CI_WASM_BUILD_EMSDK_PATH" ]]; then
+             source $NATIVEAOT_CI_WASM_BUILD_EMSDK_PATH/emsdk_env.sh
+        fi
```

Comment:
The Windows equivalent is in `eng/common/build.ps1`. Can we do the same with `eng/common/build.sh` to keep the scripts similar?

Reply:
Yes, moved.